package.json

```diff
@@ -64,6 +64,9 @@
     "webpack": "^5.38.1",
     "webpack-cli": "^4.7.2"
   },
+  "browser": {
+    "crypto": false
```

Comment:
What is the purpose of this? Right now I think it may be causing some problems, since the key `browser` is also defined before this

Reply:
This is an ugly fix for the status quo, if we do not disable the `crypto` field, then the Travis test in browser, and local test in browser, will both fail.

We are not quite sure of a clear solution. I believe the current fix comes from: https://stackoverflow.com/questions/54162297/module-not-found-error-cant-resolve-crypto/54162447.

Personally both Andrew and I do not like duplicate the same field in json config, any thoughts on that?

Reply:
This appears to be an issue with the test script, not with the build in general, since `npm run build` still works.

I think the actually issue is a circular dependency somehow. I made the change from https://github.com/algorand/js-algorand-sdk/pull/454#discussion_r742418470 and changed the import line in `10.ABI.ts` from `import algosdk from '..';` to `import algosdk from '../index';`, and now `npm test` works properly now without this `package.json` change. @algochoi let me know if this works for you as well.

Reply:
I think it works locally too -- I changed all the `import algosdk` dependencies and just pushed the changes 😃